test/core_dom/resource_url_resolver_spec.dart

```diff
@@ -241,6 +241,50 @@ void main() {
     _run_resolver(useRelativeUrls: true);
     _run_resolver(useRelativeUrls: false);
   });
+
+  describe('url_resolver uri rewrite', () {
+    ResourceUrlResolver resolver;
+
+    beforeEach((){
+      var config = new ResourceResolverConfig.resolveRelativeUrls(true);
+      var typeMapper = new DynamicTypeToUriMapper();
+      resolver = new ResourceUrlResolver.forTests(typeMapper, config, 'http://localhost');
+    });
+
+    it('should not rewrite absolute URIs ', () {
```

Comment:
technically empty sting is not absolute URI, so I would say `absolute paths, empty paths and custom schemas`.